### DISCUSSION TITLE
关于 ShokaX 向 Valaxy 迁移过程中功能增删改进的提案&对 ShokaX Valaxy 版本的一些建议 [代发]

### DISCUSSION THREAD

**Discussion by zkz098** [theme-shoka-x/hexo-theme-shokaX#166] [Ideas]
> 本讨论原作者受制于网络问题无法在github发起讨论，由本人(zkz098)代发，正文如下：
# 关于 ShokaX 向 Valaxy 迁移过程中功能增删改进的提案&对 ShokaX Valaxy 版本的一些建议

## 短推文模块

短推文模块设计理念类似于 Twitter、Bilibili 的“动态”、微信的“朋友圈”功能，直接将所有短推文内容展示在主页或特定页面内。该功能旨在用户需要发布短文章时能够将短文章集中起来，可以使网站变得更加简洁。

* 假设用户甲是一名诗人兼 Blogger，其可能会在自己的博客上发布大量短诗，此时如果按普通推文发布将会有进入推文页面的时间成本消耗与推文长度不成正比的情况发生，会削弱用户体验。
* 假设用户乙热衷于分享生活，其可能会在自己的博客上晒出大量生活写真，此时如果按普通文章发布将会显得臃肿。

## 网站 Title

ShokaX 现版本中，网站 Title 会随着标签页状态（活跃或隐藏）而改变，但事实证明，改变网站 Title 的功能不会有足够数量的人设置，所以建议将网站 Title 变为常值。

## 加载动画

因为 Valaxy 架构性能十分优越，加载动画不会有太长的展示时间，会占用资源。出于性能及资源利用考虑，建议部分取消相关动效，仅保留基础页面。

## 自定义 UI 样式
     
ShokaX 现版本中，UI 界面高亮色大多为橙 - 粉渐变，出于个性化的需求，可以在配置文件里加入UI设置相关内容（方角按钮等）。

**Comment by zkz098** [comment]
我的一位朋友提供的建议（）
@D-Sketon 看看是否需要在valaxy版中实现或做修改，这个建议应该没有大问题（）

**Comment by D-Sketon** [comment]
加载动画更多的是作用于网络波动而非框架性能

**Reply by zkz098** [reply]
已通知修改。
后续更新版本直接看 https://app.simplenote.com/publish/6J0J0J 就行了，不当大自然的搬运工了（）

**Comment by D-Sketon** [comment]
是否需要提供埋点接口，以方便那些需要自行托管统计数据的用户？

**Reply by zkz098** [reply]
显然有（）

**Comment by D-Sketon** [comment]
是否可以考虑采用一种智能的降级模式:在低版本内核或是移动设备上部分移除甚至完全移除高消耗的动效？

**Reply by zkz098** [reply]
我觉得可以
对低版本内核可能不需要那么仁慈，可以直接掐掉（）
